README.md

```diff
@@ -3,23 +3,27 @@
 
 This repo contains code used to run the Monash Human Power camera system. It is write protected to ensure it is always left in a working state.
 
-## Important-changes log
-  - Added `requirements.txt` to work with virtualenv (see Dependencies below)
-  - Code for V1.5 is in `Legacy` folder, code for V2 is in `CameraOverlay` folder
-  - Refactor GLOBAL_DATA -> DAS_DATA and REQUIRED_DATA -> POWER_MODEL_DATA
-
-## Dependencies
-To install Pillow (PIL fork), RPi.GPIO, paho-mqtt on Raspberry Pi: **haven't tested on fresh Raspberry Pi**
-  - install virtual env `sudo pip3 install virualenv`
-  - install system dependencies listed in Pillow's [doc](https://pillow.readthedocs.io/en/latest/installation.html#linux-installation) `sudo apt-get install libtiff5-dev libjpeg8-dev zlib1g-dev libfreetype6-dev liblcms2-dev libwebp-dev`
-  - in MHP_Raspicam folder, run 
-    ```
-    virtualenv venv
-    source venv/bin/activate   
-    ````
-  - install python dependencies `pip3 install -r requirements.txt`
-  - have an ```.env``` file in the ```CameraOverlay``` folder with the variable ```MHP_CAMERA``` set to either ```primary``` or ```secondary```
-  - create an ```config.json``` file in the ```CameraOverlay``` directory with attribute ```activeOverlay``` that points to the overlay file that is currently active, for instance it could be ```{ "activeOverlay": "overlay_all_stats.py" }```
+## Installation
+
+Firstly, you'll need to create a `.env` file containing either `MHP_CAMERA=primary` or `MHP_CAMERA=secondary`. Optionally, you may set the overlay run when the physical switch on the display is toggled by modifying `config.json`. An example of this file's contents could be `{ "activeOverlay": "overlay_all_stats.py" }`.
+
+To install dependencies, you will need [poetry](https://python-poetry.org/docs/#installation) installed on your computer. Our guide to poetry may be found on [notion](https://www.notion.so/Getting-Started-with-Poetry-770384c0205c4cf39c4bf7216060d5d2).
```

Comment:
Not sure if we should be linking notion on git. Don't need to mention the guide.